mkosi.md

```diff
@@ -792,32 +782,31 @@ a machine ID.
 : Takes a path to an executable that is used as build script for this
   image. If this option is used the build process will be two-phased
   instead of single-phased. The specified script is copied onto the
-  development image and executed inside an `systemd-nspawn` container
-  environment. If this option is not used, but the `mkosi.build` file
-  found in the local directory it is automatically used for this
-  purpose (also see the "Files" section below). Specify an empty value
-  to disable automatic detection.
+  development image and executed inside a chroot environment. If this
```

Comment:
I think we should be explicit here (and in the rest of the docs, where it says `chroot`), that we're using bubblewrap, just so that people know that the assurances are stronger than with a plain chroot.

Reply:
The thing is, bubblewrap on its own doesn't say much, since its a pretty generic tool

Reply:
That's true, although this might be a tad too vague, how about
```suggestion
  development image and executed inside a namespaced chroot environment. If this
```

It's not really ideal, since it doesn't say anything about what namespaces are being used, but reading it, will hopefully give people the idea that it's more than a simple chroot.